open-class/docs/classes/assignment-branch-logic.md

```diff
@@ -10,244 +10,187 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-
-<img alt="Yes/No question" src={useBaseUrl('img/Yes_no.png')} />
+### What do you need to know?
 
-Docassemble uses two different kinds of `question` blocks. This new kind only lets you ask for one variable on a screen, and doesn't use the `fields` specifier we learned earlier. It's kind of a simplified shorcut version.
+Well, without user research, you don't really know what good questions would be. You don't have funding or time to do that discovery work, though, but you don't care that much about those strangers and it's not like they have other options, so you'll start by making a guess and worry about it later.
 
-Let's look at line 3 of our sample:
+Do they even like parties? That seems like a place to start.
 
 ```yaml
-yesno: over_eighteen
+---
+id: does_like_parties
+question: |
+  Do you like parties?
+fields:
+  - Do you like parties: user_likes_parties
+---
 ```
+<!-- I'd stick with fields for now because those will be used most consistently -->
 
-This is a new specifier: the `yesno` specifier. It always shows the text `yes` and `no` as buttons on the screen. On the right hand side of the : is the variable that will be assigned the value `True` or `False`, depending on which button the user clicks.
-
-### Planning out our variables
-
-It can help to start out by writing a list of the actual variables we will want to ask. This way, we can plan variable names that make sense together.
-
-In Docassemble interviews, it's traditional to use variables names that are:
+Docassemble will show two buttons on this question screen. If the user clicks 'Yes', `user_likes_parties` will have a value of `True`. If they click 'No', it will have a value of `False`. We can use that to help our program decide what to do. [Read more about yes/no fields](https://docassemble.org/docs/fields.html#fields%20yesno).
 
-1. Short
-1. Descriptive (without cryptic abbreviations)
-1. Lower-case
-1. Use _ to separate multiple words when needed
+### If this, then that
 
-We are going to need true/false variables and an ending screen. Let's name them as follows:
+We'll use what's called an `if` statement. It tells our program, 'if the situation is this, then do that.'
 
-variable name | meaning
---------------|---------
-go_to_party | A screen that tells the person to go to the party
-dont_go_to_party | A screen that says not to go to the party
-likes_parties | Do you like parties?
-wants_to_meet | Do you want to meet someone there?
-big_assignment | Do you have a big assignment due?
-
-### Writing our first question blocks
-
-Let's start with writing the ending blocks, which are the simplest. Remember, we will use the `event` specifier for these because they are ending screens.
-
-```yaml
+``` yaml
+---
+id: flow
+mandatory: True
+code: |
+  if ( user_likes_parties ):
+    go_to_party
+  else:
+    dont_go_to_party
 ---
-event: go_to_party
-question: |
-  You should go to the party!
-subquestion: |
-  [FILE https://media.giphy.com/media/l0MYt5jPR6QX5pnqM/giphy.gif]
```

Comment:
Note: This is fun, but it adds another concept. If I feel I've done a really good jobs keeping things simple in the rest of the lesson, I might splurge and include something like this here. It'd be a hard choice.

Reply:
Also, though, I find gifs a bit annoying if I'm trying to deeply read the text around them, so I might not pick a gif, or maybe not include it in the lesson page itself and, instead, link to an interview screen that shows it 🙃

Reply:
Btw, are they familiar with `subquestion`?

Reply:
Yep, they have used subquestion. The stretch from Assignment 1 was to embed an image. I chose this in part because of reading that including images/videos early on in assignments for coding was a useful way to help new learners. But I think it's not needed in the actual assignment text, so I'll take it out there.

Reply:
Actually, if you folks learned it in an earlier less, I'd say have it in here too. Repetition is good for learning 👍